src/test/java/org/takes/tk/TkSlf4jTest.java

```diff
@@ -55,4 +61,36 @@ public void logsException() throws IOException {
         new TkSlf4j(new TkFailure(new IOException(""))).act(new RqFake());
     }
 
+    /**
+     * {@link TkSlf4j} can output an empty body for {@link TkEmpty}.
+     * @throws IOException if some I/O problem occurred.
+     */
+    @Test
+    public void testEmptyResponseBody() throws IOException {
+        new FtRemote(
+            new TkSlf4j(new TkEmpty())
+        ).exec(
+            new FtRemoteScriptPost()
+        );
+    }
+
+    /**
+     * {@link FtRemote.Script} that does a POST request with a "test" body.
+     * @author Rui Castro (rui.castro@gmail.com)
+     * @version $Id$
+     * @since 0.43
+     */
+    private class FtRemoteScriptPost implements FtRemote.Script {
```

Comment:
@essobedo originally it was an anonymous class, but Qulice complained that `TkSlf4jTest` had to much data coupling (it was using too many other classes, 10 or 11, when the maximum was 7). I moved that anonymous class to a different class to reduce that number and pass Qulice control.